config/deploy.js

```diff
@@ -101,6 +101,12 @@ function deployBuild(url, folder) {
         let next_path = path.resolve(folder);
         shell.cd(next_path);
         const start = Date.now();
+        if (url === EDITOR_URL) {
+            shell.exec('git clone ' + url + ' current_build');
+            shell.exec('\cp -f current_build/*.js current_build/*.css .');
```

Comment:
Had it because I saw online that 'cp' is often aliased to 'cp -i', but after checking, this does not seem to be an issue for us. Fixed.